render_delegate/material.h

```diff
@@ -76,7 +76,7 @@ class HdArnoldMaterial : public HdMaterial {
     ///
     /// Currently does nothing.
     HDARNOLD_API
-    void Reload() {}
+    void Reload() override {}
```

Comment:
I remember now, this was marked non-override on purpose. Newer USD versions don't provide this function anymore. It is missing in 20.11 (https://github.com/PixarAnimationStudios/USD/blob/v20.11/pxr/imaging/hd/material.h) but = 0 in 20.08 https://github.com/PixarAnimationStudios/USD/blob/v20.08/pxr/imaging/hd/material.h. We can wrap in #if PXR_VERSION < 2011 , that should solve your warning and still work with newer USD versions.